_security/access-control/api.md

```diff
@@ -1297,6 +1297,85 @@ PATCH _plugins/_security/api/securityconfig
 }
 ```
 
+### Check configuration upgrade
+Introduced 2.13
+{: .label .label-purple }
+
+Checks the configuration bundled with the Security plugin against the existing security configuration, returns if an upgrade can be performed and what resources would be updated.
```

Comment:
I think we should add a blurb on why someone would use this endpoint.

wdyt of this?

OpenSearch adds new functionality between releases. When a new plugin is added to the default bundle, it also includes reserved roles for cluster administrators to map to users to permit usage of the new functionality. This endpoint helps cluster administrators check if the cluster is missing default roles or has stale definitions of default roles.

Reply:
Added a note for this section

Reply:
![image](https://github.com/opensearch-project/documentation-website/assets/2754967/77158de3-e787-4186-87f3-441c6a57cffc)

How this looks rendered out